app/wss.js

```diff
@@ -0,0 +1,50 @@
+const { WEBSOCKET_PORT } = require("../env.json");
+const { WebSocketServer } = require('ws');
+const { getCurrentRoom, enterRoom } = require("./game/human.js");
+const { HumanRoomUpdateData } = require("./dataObjects.js");
+
+
+const webSockServer = new WebSocketServer({ port: WEBSOCKET_PORT });
+
+const onConnectionClose = () => {
+    console.log("client disconnected!");
+};
+const onError = () => {
+    console.log("websocket error");
+}
+const sendToAllClients = (data) => {
+    if (typeof data === "object") {
+        data = JSON.stringify(data);
+    }
+    webSockServer.clients.forEach(client => {
+        client.send(data);
+    });
+}
+const onReceiveDataFromClient = (byteData) => {
+    let data = JSON.parse(byteData.toString());
+    let action = data.action;
+    let currentRoom;
+    switch(action.name) {
+        case "enterRoom":
+            enterRoom(action.args.room);
+            currentRoom = getCurrentRoom();
+            sendToAllClients(HumanRoomUpdateData(currentRoom));
+            break;
+        case "getCurrentRoom":
+            currentRoom = getCurrentRoom();
+            sendToAllClients(HumanRoomUpdateData(currentRoom));
```

Comment:
If a client asks to get the human's room, it only needs to be sent to the client who asked for it rather than all clients

Reply:
Right now there's no way to discern a request from the ai player and human player, but there is logic in play now with this ticket by using the ticket, but haven't refactored everything to use this now, ideally I'll shove it in another PR